### PR TITLE
[NS2.1] Import some primitive System.Drawing types from CoreFX

### PR DESCRIPTION
**Pull request by EgorBo**
Contributes to https://github.com/mono/mono/issues/11430

### CODE REVIEW

File: mcs/class/corlib/corlib_xtest.dll.sources [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 ../../../external/corefx/src/System.Reflection/tests/Common.cs
 
-../../../external/corefx/src/System.Reflection/tests/*.cs:AssemblyTests.cs,AssemblyNameTests.cs,MethodInfoTests.cs,ModuleTests.cs
+../../../external/corefx/src/System.Reflection/tests/*.cs:AssemblyTests.cs,TypeInfoTests.netcoreapp.cs,AssemblyTests.netcoreapp.cs,AssemblyNameTests.cs,MethodInfoTests.cs,ModuleTests.cs
```
Comment (marek-safar) [inline mcs/class/corlib/corlib_xtest.dll.sources]: Why is this change needed?

Comment (EgorBo) [reply mcs/class/corlib/corlib_xtest.dll.sources]: @marek-safar "temp" fix for the changes I pulled with "bump corefx". Will get rid of it once Maxim merges his PR

File: mcs/class/System.Drawing/System.Drawing.dll.sources [hunk 2]
```diff
@@ -27,8 +27,10 @@ System.Drawing/BufferedGraphicsManager.cs
 System.Drawing/BufferedGraphicsContext.cs
 System.Drawing/CharacterRange.cs
 System.Drawing/macFunctions.cs
-System.Drawing/ColorConverter.cs
```
Comment (marek-safar) [inline mcs/class/System.Drawing/System.Drawing.dll.sources]: Please remove the files as well

**Comment by EgorBo** [comment]
@monojenkins build failed

**Comment by marek-safar** [comment]
@monojenkins build failed

**Comment by marek-safar** [comment]
@monojenkins commit csproj and apidiff
